app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::GreetingsController < ApplicationController
+  def index
+    @greetings = Greeting.all
+    render json: @greetings, status: :ok
```

Comment:
- [ ] Great @GhausAliKhan, you are very close to getting project approval, as per the project requirements, instead of returning the list of all `greetings`, please return only one `random` greeting. Kindly adhere to the project requirements and implement the things accordingly.